tests/Core/List/FluentPersonaTests.cs

```diff
@@ -29,14 +29,46 @@ public void FluentPersona_Image()
         cut.Verify();
     }
 
+    [Theory]
+    [InlineData("Denis Voituron")]
+    [InlineData("Denis    Voituron")]
+    public void FluentPersona_Initials(string name)
+    {
+        // Arrange
+        var cut = TestContext.RenderComponent<FluentPersona>(parameters =>
+        {
+            parameters.Add(p => p.Id, "myComponent");
+            parameters.Add(p => p.Name, name);
+            parameters.Add(p => p.DismissTitle, "Remove this people");
+        });
+
+        // Assert
+        cut.Verify();
```

Comment:
Using `Theory` attribute, you should use `Verify(suffix:` to identify the generated html files.

Reply:
I fixed it.